src/redux/books/books.js

```diff
@@ -0,0 +1,18 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] As a good practice, let's scope our action types' names based on the feature they belong to. This will help you in the future if you decide to scale your app. You will need to do something of this sort: `const ADD_BOOK = 'bookstore/books/ADD_BOOK';`